10 add interactive map with kebab information to admin panel

Wiktor-Wojciechowski opened the pull request:
This should close #10 

- Added an interactive map with kebab information
- Made small fixes to other pages

Please check if everything is okay

Wiktor-Wojciechowski committed:
added a leaflet map

Wiktor-Wojciechowski committed:
Merge branch 'main' of https://github.com/MatWisie/KebabFinder-API into 10-add-interactive-map-with-kebab-information-to-admin-panel

Wiktor-Wojciechowski committed:
map now displays pins placed at kebab locations, pins can display details about kebabs

Wiktor-Wojciechowski committed:
added comments placeholder to kebabpanel

Wiktor-Wojciechowski committed:
Merge branch 'main' of https://github.com/MatWisie/KebabFinder-API into 10-add-interactive-map-with-kebab-information-to-admin-panel

Wiktor-Wojciechowski committed:
request path update

Wiktor-Wojciechowski committed:
reduced code redundancy, show more button now opens kebab panel

MatWisie reviewed (changes requested):
1. In kebab list last item has "Show More" button overflow
![obraz](https://github.com/user-attachments/assets/e9f73a88-f5c6-4a82-83da-20e07881bcc8)

2. Shouldn't map be visible only for auth admin? After all its admin panel.

Wiktor-Wojciechowski committed:
made index route protected

Wiktor-Wojciechowski committed:
fixed isfirstlogin not getting a token, made navbar disappear when not logged in

Wiktor-Wojciechowski committed:
fixed kebab list height

MatWisie reviewed (changes requested):
All right, this pull request will be approved after https://github.com/MatWisie/KebabFinder-API/pull/30 has been merged as I need to check it with newest possible comments controller

Wiktor-Wojciechowski committed:
Merge branch 'main' of https://github.com/MatWisie/KebabFinder-API into 10-add-interactive-map-with-kebab-information-to-admin-panel

Wiktor-Wojciechowski committed:
typo fix in swagger docs

Wiktor-Wojciechowski committed:
fixed route to kebab comments

Wiktor-Wojciechowski committed:
corrected the paths in kebab section of swagger docs

Wiktor-Wojciechowski committed:
added a route to get a user by id, added docs

Wiktor-Wojciechowski committed:
comments now return user data, display changes

W-Worach commented on KebabBackend/routes/api.php:
This route should only for admin because it returns informations that ordinary user shouldn't see (i.e. email address, created_at and etc)

W-Worach commented on KebabBackend/app/Services/CommentService.php:
Same situation there i dont think everyone should see all user informations, just the name and is_admin should be enought

W-Worach commented:
![image](https://github.com/user-attachments/assets/bab84021-5faa-4b5f-b47f-70804b1eefc0)
Also I would suggest to make categories (opening hours, souces, meat types etc) to make text bold to highlight them from their informations

Wiktor-Wojciechowski committed:
changed comment layout, added deleting comments

Wiktor-Wojciechowski committed:
adjusted summary styles for kebab panel

W-Worach committed:
changed route for getting informations about user

changed route for getting informations about user to match route from all users
also now this route is required to have admin role

W-Worach committed:
changed get kebabs comments

changed get kebabs comments to not give sensitive informations

W-Worach committed:
added additional informations to get user comments

now get user comments gives same informations as get kebab comments

Wiktor-Wojciechowski committed:
added admin/user indicator, adjusted kebab panel to look better on smaller devices
